test/docs/__snapshots__/get-docs-yaml.api-test.ts.snap

```diff
@@ -808,9 +808,13 @@ components:
         companyName:
           type: string
           description: Company Name
+        probabilityOfDefault:
+          type: number
+          description: Probability of Default
```

Comment:
Is this missing LGD and customer rating?

Reply:
Ah thanks I'll regenerate